trail/trail/urls.py

```diff
@@ -18,6 +18,22 @@
 from django.conf import settings
 from django.urls import path, re_path, include
 from django.views.generic import TemplateView
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework import permissions
+
+schema_view = get_schema_view(
+   openapi.Info(
+      title="TrailBlazer API",
```

Comment:
I feel like these keyword args would best be punted to the Config class which would read them from a file. See [this](https://github.com/dirac-institute/trailblazer/blob/main/trail/trail/settings.py#L31) and [this](https://github.com/dirac-institute/trailblazer/blob/main/trail/trail/config.py) for the Config class and [this](https://github.com/dirac-institute/trailblazer/tree/main/trail/config) for example of the files. 

None of these values are secrets so you can add them directly to the config directory. 

License information is in our repo, see [here](https://github.com/dirac-institute/trailblazer/blob/main/LICENSE)

Description could be something better than "Test PR" such as "Rest API Documentation", make sure we change this can stay like that but we should put an admin email there as soon as we manage.